src/renderer/features/fellowship-tasks/model/tasks.ts

```diff
@@ -43,6 +43,7 @@ const $evidencePeriods = fellowship.$store.map(store => store?.evidencePeriods ?
 const $maxRank = fellowship.$store.map(input => input?.maxRank ?? 0);
 const $members = fellowship.$store.map(input => input?.members ?? []);
 const $chainName = $chain.map(chain => chain?.name ?? 'Unknown');
+const $referendumsWithEvidence = fellowship.$store.map(store => store?.referendumsWithEvidence ?? []);
```

Comment:
same here, do we have an another mechanism of distinguishing suspense from emptiness? if so then forget